src/Resources/contao/library/Contao/User.php

```diff
@@ -588,22 +571,7 @@ public function setSalt($salt)
 	 */
 	public function getEncoderName()
```

Comment:
We should remove this method and the `EncoderAwareInterface`

Reply:
Removed in 01bb96ee64808027bb6bc15d5cdc02911d5f9d8b.